seatunnel-connectors-v2/connector-cdc/connector-cdc-oracle/src/main/java/org/apache/seatunnel/connectors/seatunnel/cdc/oracle/source/eumerator/OracleChunkSplitter.java

```diff
@@ -104,21 +103,4 @@ protected int ObjectCompare(Object obj1, Object obj2) {
             return ObjectUtils.compare(obj1, obj2);
         }
     }
-
```

Comment:
Why does he need to be removed? I think choosing it in the absence of a primary key is also a valid option.

Reply:
If you select this field as the split column, output the column to the sink.

Reply:
ok,In a certain scenario, there can be a bug with the ROWID when a table has existing data before enabling logging, causing the inability to read snapshots.